ansible_base/common/models/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .organization import AbstractOrganization
```

Comment:
Lets rename "models" to "abstract_models" to make it clear that this isn't the same as the other django models modules.